docs/chapters/17-the-uncarved-bock/index.html

```diff
@@ -469,6 +469,13 @@ <h1 class="chapter-title"><span>The</span> Uncarved Block</h1>
     </blockquote>
   
 
+  
+    <img
+      src="/assets/images/chapter/preacher-pale-rider.jpg"
+      alt=""
```

Comment:
I know this is the compiled file, but I see we have empty alt tags, and I think that makes sense with the SVG since they are decoration, but don't these images tell more of a story?

Reply:
None of them contain particular information that would be necessary for the reader – they are more presentation than content.